pkg/operator/controller/ingress/controller.go

```diff
@@ -1277,7 +1284,34 @@ func computeUpdatedInfraFromService(service *corev1.Service, infraConfig *config
 	if platformStatus == nil {
```

Comment:
Can you update the GO Docs for `computeUpdatedInfraFromService` to reflect that it also AWS now:
```
// computeUpdatedInfraFromService updates GCP's PlatformStatus with Ingress LB IPs when the DNSType is `ClusterHosted`.
```

Reply:
Done.